forestry/api/genetics/IFlowerProvider.java

```diff
@@ -11,17 +11,11 @@
 import net.minecraft.world.World;
 
 public interface IFlowerProvider {
-	/**
-	 * @return True if the block at the passed coordinates is a valid flower for the species.
-	 */
-	boolean isAcceptedFlower(World world, IIndividual individual, int x, int y, int z);
-
-	boolean isAcceptedPollinatable(World world, IPollinatable pollinatable);
 
 	/**
-	 * @return True if a flower was planted.
+	 * @return Unique identifier for this type of flower.
 	 */
-	boolean growFlower(World world, IIndividual individual, int x, int y, int z);
+	String getFlowerType();
```

Comment:
"Unique identifier for this type of flower"?
Shouldn't it be "Unique identifier for this type of flower provider"?